src/jgikbase/idmapping/core/errors.py

```diff
@@ -12,43 +13,46 @@ class ErrorType(Enum):
     :ivar error_type: a brief string describing the error type.
     """
 
-    AUTHENTICATION_FAILED =  (10000, "Authentication failed")  # noqa: E222 @IgnorePep8
+    AUTHENTICATION_FAILED = (10000, "Authentication failed")  # noqa: E222 @IgnorePep8
     """ A general authentication error. """
 
-    NO_TOKEN =               (10010, "No authentication token")  # noqa: E222 @IgnorePep8
+    NO_TOKEN = (10010, "No authentication token")  # noqa: E222 @IgnorePep8
     """ No token was provided when required. """
 
-    INVALID_TOKEN =          (10020, "Invalid token")  # noqa: E222 @IgnorePep8
+    INVALID_TOKEN = (10020, "Invalid token")  # noqa: E222 @IgnorePep8
     """ The token provided is not valid. """
 
-    UNAUTHORIZED =           (20000, "Unauthorized")  # noqa: E222 @IgnorePep8
+    UNAUTHORIZED = (20000, "Unauthorized")  # noqa: E222 @IgnorePep8
```

Comment:
This section more readable when the error codes are all whitespace aligned. I think the noqa should stop fake from complaining

Reply:
👍